config/locales/jobseekers/job_application/personal_details_form.yml

```diff
@@ -100,11 +126,25 @@ en:
           true: Yes, I've completed this section
         working_pattern_details: Provide more details about your working patterns preference (optional)
         working_patterns: Full, part time or job share
+      jobseekers_uploaded_job_application_personal_details_form:
+        email_address: Email address
+        first_name: First name
+        last_name: Last name
+        phone_number: Phone number
+        personal_details_section_completed_options:
+          false: No, I'll come back to it later
+          true: Yes, I've completed this section
+      jobseekers_uploaded_job_application_upload_application_form_form:
+       upload_application_form_section_completed_options:
+          false: No, I'll come back to it later
+          true: Yes, I've completed this section
```

Comment:
Question (I may be lost here). Are these translations used at all?
The view seems to be calling the `helpers.label.jobseekers_job_application_personal_details_form.first_name` labels rather than these ones.

And it works as they're the same wording. We could just refer to them or just use the others. Don't know the best practice on that.

Reply:
We use these in app/views/jobseekers/uploaded_job_applications/personal_details/edit.html.slim